### ISSUE TITLE
Provide the generated AppImage for download 

### ISSUE THREAD

**Issue by probonopd** [jvaltin/redesigned-invention#3]
https://github.com/jvaltin/redesigned-invention/blob/85500a1f09873966a64947343f53ca1206eabe59/Makefile#L67

Please provide the generated AppImage for download on GitHub Releases, thank you. This way we could include it in the AppImageHub central directory of available AppImages.

**Comment by benma** [comment]
You already are in the process of adding it: https://github.com/AppImage/appimage.github.io/pull/762

**Comment by jvaltin** [comment]
@probonopd  Is there a specific Makefile target you would suggest for uploading the built AppImage?

**Comment by probonopd** [comment]
No, as I usually do the AppImage building in bash scripts or `.travis.yml` files.

**Comment by jvaltin** [comment]
Is there documentation you would suggest on the topic?

**Comment by probonopd** [comment]
https://docs.appimage.org/ - please let me know if you are missing something there. Thanks!
